docs/UserGuide.adoc

```diff
@@ -227,6 +227,35 @@ Format: `exit`
 Address book data are saved in the hard disk automatically after any command that changes the data. +
 There is no need to save manually.
 
+== Features(Coming in v2.0)
```

Comment:
I may not have been clear enough, but `v2.0` is the sub-section for features which we do not intend to implement within the scope of CS2103. Please shift these features which we do intend to implement to the correct section (i.e. the general `Features` section)

Reply:
My interpretation of the course documents is that all features not yet implemented are marked as `Coming in v2.0` until they are implemented, after which you mark them as `Since [version number]`. Is this incorrect?

Reply:
yup sorry, my bad